### ISSUE TITLE
Regression with multiple variables

### ISSUE THREAD

**Issue by GustafRydevik** [gertvv/gemtc#51]
Hi,

The manual seem to indicate that it is currently only possible to include a single variable in a network meta regression. Is this correct? If so, do you have any pointers for how to update the code to allow for multiple variables>

**Comment by gertvv** [comment]
That is correct. You'd need to have a pretty good understanding of the overall inner workings of the package before attempting this.
